parsing/dialects/intermediate/dialects/dialect1/keywords.yaml

```diff
@@ -1,4 +1,4 @@
-# Licensed to the Apache Software Foundation (ASF) under one or more
+ # Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
nit: Remove extra space

Reply:
Fixed in https://github.com/googleinterns/calcite/pull/245/commits/01b6e805a3c5c8a60af83236b5e706211afdff7c